tracks/go/exercises/raindrops/README.md

```diff
@@ -0,0 +1,65 @@
+# Raindrops
+
+This is the third core exercise in the go track to receice feedback from a mentor. It is less about learning new concepts but more about deepening the already acquired knowledge. To some the conversion of an `int` to `string` and the modula operator `%` are new.
+
+This may be an excellent place to talk about the go tooling again -- especially if the tools were not applied (`gofmt`, `golint`, etc).
+
+## Reasonable solutions
+
+The exercise can be solved with `if` statements and `string` concatenation:
+
+
+```
+// Convert implements raindrop speech
+func Convert(i int) string {
+	var res string
+	if i%3 == 0 {
+		res += "Pling"
+	}
+	if i%5 == 0 {
+		res += "Plang"
+	}
+	if i%7 == 0 {
+		res += "Plong"
+	}
+
+	if res == "" {
+		res = strconv.Itoa(i)
+	}
+
+	return res
+}
+```
+
+## Common suggestions
+
+The most common feedback revolves around:
+
+* General programming things (working with strings, map iteration)
+* Readability (formatting, comments, linting)
+
+**Map: random iteration** 
+* are they defining static values (map, slice) in the function? Suggest moving them to package level.
+* are they using a `map` to hold the cases? This gets complicated as they need to somehow make sure the order is correct. From the [specs](https://golang.org/ref/spec#For_statements): "The iteration order over maps is not specified and is not guaranteed to be the same from one iteration to the next." Suggest using 3 `if` statements instead. If they prefer a loop you could direct them towards using: `for i := 3; i < 8; i += 2`
```

Comment:
To this point, switch case might be a better option instead of 3 if?

Reply:
A `switch` would be valid inside a for loop which is what I suggest if they prefer a loop.

Not sure either what the best help is in this case. If I should nudge them towards the simplest (and almost fastest) solution (3 if statements -- see section "Reasonable Solution") or stick closer to what they have done.

Reply:
I think either solution would be quite acceptable so early in the track

Reply:
so I guess stick to what is closer to the submitted solution